### ISSUE TITLE
linux: misleading indent or bug

### ISSUE THREAD

**Issue by kthelgason** [libuv/libuv#749]
The following lines in `src/unix/linux-core.c` introduced in 6798876a6b2c2046d926bd2317284de920e6d124 exhibit a classic issue

```c
599:  if (err == 0)
600:    rewind(statfile_fp);
601:    err = read_times(statfile_fp, numcpus, ci);
```

The statement on line 601 will be executed regardless of whether `err == 0`. This may be intentional but in that case the indent is very misleading.
cc: @ljagiello @saghul

**Comment by saghul** [comment]
@kthelgason good catch. There should be braces areound that if. We only want to execute that if there was no error, that way, err won't be overwritten and the cleanup code will take care of it. Do you want to send a PR?

**Comment by kthelgason** [comment]
@saghul Yeah I figured this was not the intention. :smiley: I've opened a PR to fix this.

**Comment by bnoordhuis** [comment]
Fixed in 80d9d86.
